src/assets/scripts/client/InputController.js

```diff
@@ -446,30 +360,29 @@ export default class InputController {
                 this.onCommandInputChangeHandler();
 
                 break;
-            case KEY_CODES.ENTER:
-                this.input_parse();
-
-                if (this.input_run()) {
+            case KEY_CODES.ENTER: {
+                if (this.commandBarContext === COMMAND_CONTEXT.AIRCRAFT) {
```

Comment:
can we genericize this `case` a little?
```javascript
case KEY_CODES.ENTER {
    this.processCmmand();
    this.selecteAircraft();

    break;
}
```

move the `if/else` to  `.processCommand()` and then get rid of the `else` with an early return.  That makes all of this a whole lot easier to test and easier to read.

Reply:
Love it! Done.